okhttp/src/main/kotlin/okhttp3/Dns.kt

```diff
@@ -27,7 +27,7 @@ import okhttp3.Dns.Companion.SYSTEM
  *
  * Implementations of this interface must be safe for concurrent use.
  */
-interface Dns {
+fun interface Dns {
```

Comment:
Just checking, this I believe is safe, just changes the kotlin metadata in the class file, no breaking change here to existing clients.